app/filtering.js

```diff
@@ -31,36 +32,44 @@ function registerForSession (session) {
     }
 
     let results
-    let cbArgs
     for (let i = 0; i < filteringFns.length; i++) {
       let currentResults = filteringFns[i](details)
       if (currentResults && !module.exports.isResourceEnabled(currentResults.resourceName)) {
         continue
       }
       results = currentResults
-      cbArgs = cbArgs || results.cbArgs
```

Comment:
I got rid of this because `results.cbArgs` seemed to always be undefined or `{}`